notification.protocol.yaml

```diff
@@ -0,0 +1,104 @@
+
+severity_levels:
+  - id: "unknown"
+  - id: "info"
+  - id: "low"
+  - id: "medium"
+  - id: "high"
+  - id: "critical"
+
+telegram_channels:
+  - id: "Telegram1"
+    description: "Telegram channel for debug messages (severity: unknown, no quorum)"
+    token: "YOUR_TELEGRAM_BOT_TOKEN_1"
+    chat_id: "YOUR_CHAT_ID_1"
+  - id: "Telegram2"
+    description: "Telegram channel for all messages >= low, no quorum"
+    token: "YOUR_TELEGRAM_BOT_TOKEN_2"
+    chat_id: "YOUR_CHAT_ID_2"
+  - id: "Telegram3"
+    description: "Telegram channel for messages >= low, with quorum"
+    token: "YOUR_TELEGRAM_BOT_TOKEN_3"
+    chat_id: "YOUR_CHAT_ID_3"
+  - id: "Telegram4"
+    description: "Telegram channel for high and critical messages, with quorum"
+    token: "YOUR_TELEGRAM_BOT_TOKEN_4"
+    chat_id: "YOUR_CHAT_ID_4"
+
+discord_channels:
+  - id: "Discord1"
+    description: "Discord channel for debug messages (severity: unknown, no quorum)"
+    webhook_url: "YOUR_DISCORD_WEBHOOK_URL_1"
+  - id: "Discord2"
+    description: "Discord channel for all messages >= low, no quorum"
+    webhook_url: "YOUR_DISCORD_WEBHOOK_URL_2"
+
+opsgenie_channels:
+  - id: "OpsGenie1"
+    description: "OpsGenie channel for high and critical messages"
+    api_key: "YOUR_OPSGENIE_API_KEY_1"
+
+teams:
+  - name: "protocol"
+    nats_subject: "nats.protocol.findings"
```

Comment:
it's different now `findings.nats.${ctx.config.team}`

Reply:
Yes, I 've changed topics mask